pkg/controller/core/indexer/indexer.go

```diff
@@ -110,6 +112,11 @@ func IndexWorkloadRuntimeClass(obj client.Object) []string {
 	return nil
 }
 
+// TODO: Is there a "standard" implementation of something like this??
```

Comment:
I'm not aware of such function, not need to keep the TODO: https://github.com/kubernetes/community/blob/master/sig-scheduling/CONTRIBUTING.md#best-practices.